R/decks.R

```diff
@@ -242,11 +242,11 @@ setMethod("subtitles<-", "CrunchDeck", function(x, value) {
 #' @param deck A CrunchDeck
 #' @param file The file path to save the exported deck
 #' @param format Either `"xlsx"`, `"pptx"`, or `"json"`
-#'
+#' @param ... Further options to be passed on to the API
 #' @return  the filename (`file`, if specified, or the the autogenerated file
 #' name).
 #' @export
-exportDeck <- function(deck, file, format = c("xlsx", "pptx", "json")) {
+exportDeck <- function(deck, file, format = c("xlsx", "pptx", "json"), ...) {
```

Comment:
need a query (and probably viz_specs) helper func so they can construct these somewhat intuitively